Controller/home.py

```diff
@@ -116,7 +117,6 @@ def add_new_application():
         error = "This job application could not be stored in the database. Please try again."
         return render_template('home.html', jobAddError=error)
     data = {}
-    s_email(company_name,location,job_profile,salary,username,password,session['email'],security_question,security_answer,notes,date_applied,status)
```

Comment:
Code formatting needs to be done, Please create bug for implementing black, auto black, assign @arjunptm